unreleased.md

```diff
@@ -1 +1,5 @@
 # Unreleased
+
+## Improvements
+
+- `500` exceptions retry issue, fixed.
```

Comment:
- `500` responses from running backend were causing retry instead of redirecting to error page, fixed